config.example/registry.yml

```diff
@@ -0,0 +1,29 @@
+#
+# Container registry configuration
+#
+
+# Storage
+persistence:
+  # Use persistent storage to retain data
+  enabled: true
+  # Size of persistent storage volume
+  size: 10Gi
```

Comment:
that's pretty small

Reply:
what do you want for a default? my VM only has 40Gi :)